tests/test_requests_resource.py

```diff
@@ -73,6 +98,42 @@ def test_get_a_request(self):
         response = self.client.get('/api/v1/users/request/1/')
         self.assertEqual(response.status_code, 200)
 
+        def test_get_a_non_existing_request(self):
+            """ Test user tries to retrieve a noin-existing request """
+            self.create_and_login_user()
+
+            response = self.client.post(
```

Comment:
you can have this as a function that is called to create the request whenever you need to.